### DISCUSSION TITLE
Add "mvrtype" and "value" functionality to MVT similar to MVR

### DISCUSSION THREAD

**Discussion by RyanConway91** [MODFLOW-ORG/modflow6#1314] [Q&A]
I am using MVR to route flows from a wetland complex to surface water features in a GWF/GWT model. There are geochemical processes occurring within the wetlands that reduce concentrations. I would like to account for these by reducing the portion of the solute mass routed by MVT which gets to the surface water features. Why are there not options similar to those in MVR ("mvrtype" and "value") in MVT? Could they be added?

If there another way I can only route a portion of solute mass while keeping all the water?

Thanks

**Comment by emorway-usgs** [comment]
Linked to this discussion on the [MODFLOW Development Plans Wiki](https://github.com/MODFLOW-ORG/modflow6/wiki/MODFLOW-Development-Plans).
